### PR TITLE
Tickets/dm22231

### PR DESCRIPTION
**Pull request by rbovill**
*  Updated the db_reserved list in testutils.py to the list of InfluxDB words. Added Jira skip tags to now failing tests in test_NoReservedWords.py.

### CODE REVIEW

File: tests/test_NoReservedWords.py [hunk 1]
```diff
@@ -6,7 +6,42 @@
...
+        jira = "DM-22623"
+    elif csc == "PromptProcessing" and (topic == "Telemetry" or topic == "Events"):
+        jira = "DM-22624"
+    elif csc == "Scheduler" and topic == "Telemetry":
+        jira = "DM-22625"
+    elif csc == "Script" and topic == "Events":
+        jira = "DM-22626"
+    elif csc == "Test" and topic == "Commands":
+        jira = "DM-22627"
+    elif csc == "Watcher" and (topic == "Commands" or topic == "Events"):
```
Comment (r-owen) [inline tests/test_NoReservedWords.py]: The only Watcher error I know of is in Commands (the mute command).

File: tests/test_NoReservedWords.py [hunk 2]
```diff
@@ -6,7 +6,42 @@
 
 
 def check_for_issues(csc, topic):
-    jira = ""
+    if csc == "ATAOS" and topic == "Commands":
+        jira = "DM-22612"
+    elif csc == "ATMCS" and (topic == "Commands" or topic == "Events"):
```
Comment (r-owen) [inline tests/test_NoReservedWords.py]: I am not suggesting you change the code, but in future you might consider reducing repetition using:
```
...and topic in ("Commands", "Events"):
```

File: tests/test_NoReservedWords.py (hunk 1 above)
Comment (rbovill) [reply tests/test_NoReservedWords.py]: Watcher Events uses NAME, which is also an InfluxDB Reserved Word

File: tests/test_NoReservedWords.py (hunk 2 above)
Comment (rbovill) [reply tests/test_NoReservedWords.py]: I will keep that in mind for next time.

**Pull request merged**